vowpalwabbit/kernel_svm.cc

```diff
@@ -553,15 +553,15 @@ bool update(svm_params& params, size_t pos)
   return overshoot;
 }
 
-void copy_char(char& c1, const char& c2)
+void copy_char(char& c1, const char& c2) noexcept
 {
   if (c2 != '\0')
     c1 = c2;
 }
 
-void add_size_t(size_t& t1, const size_t& t2) { t1 += t2; }
+void add_size_t(size_t& t1, const size_t& t2) noexcept { t1 += t2; }
```

Comment:
Do we have other places where we do an inplace add into the first argument? If so, can these be unified as a template?

Reply:
Yes these simple functions are scattered around. Implementing them in one common place would be good